src/roles/common/templates/vsr.j2

```diff
@@ -5,6 +5,8 @@
 
 component_type: vsr
 
+required_bridges: []
```

Comment:
Can you remove this if we are not using them

Reply:
There is a common role which checks for each component if the required bridges are present. So needed to add it here to avoid undefined variable